packages/react-core/src/components/Accordion/__tests__/AccordionContent.test.tsx

```diff
@@ -85,14 +86,14 @@ test('Renders with inherited element props spread to the component', () => {
   expect(screen.getByRole('heading')).toHaveAccessibleName('Label');
 });
 
-test('Renders with class name pf-v5-c-accordion__expandable-content', () => {
+test(`Renders with class name ${styles.accordionExpandableContent}`, () => {
```

Comment:
It _might_ impact performance, I'm honestly not sure if it would really be noticeable or not.

I think it's worth trying though, if we can't do this or something like it we'll have to update all of the classnames each major release, which isn't ideal.

Reply:
Probably minimal.  They job looks like it still completes in 4 minutes.